v23.1/crdb-internal.md

```diff
@@ -858,6 +858,244 @@ ON us.index_id = ti.index_id AND us.table_id = ti.descriptor_id
 WHERE total_reads = 0;
 ~~~
 
+### `ranges`
```

Comment:
OK. I was thinking that since folks were already using them this would be throwing them a bone for the transition.  Then again adding docs would be kind of backsliding from the current state.

Removed!